File: AudioRecord/app/src/main/java/com/example/audiorecord/activity/MainActivity.java
package com.example.audiorecord.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;

import com.example.audiorecord.R;
import com.example.audiorecord.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding mBind;
    private boolean isRecord = false;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mBind = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(mBind.getRoot());

        //设置标题
        setSupportActionBar(mBind.toolbar);
        ActionBar bar = getSupportActionBar();
        if (bar != null) {
            bar.setTitle(getString(R.string.app_name));
        }
        click();
    }

    private void click() {
        mBind.record.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if (isRecord) {
                    mBind.record.setImageResource(R.drawable.ic_record);
                    isRecord = false;
                } else {
                    mBind.record.setImageResource(R.drawable.ic_stop);
                    isRecord = true;
                }
            }
        });

        mBind.items.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, RecordListActivity.class));
            }
        });

        mBind.settings.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, SettingsActivity.class));
            }
        });
    }
}

File: part_8/app/src/main/java/com/may/part_8/MainActivity.kt
package com.may.part_8

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.may.part_8.databinding.ActivityMainBinding
import dagger.hilt.android.AndroidEntryPoint
import javax.inject.Inject

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    @Inject
    lateinit var teamManager: TeamManager

    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnGetInfo.setOnClickListener {
            val team = teamManager.getTeam()
            binding.team = team
        }


    }
}